### DISCUSSION TITLE
website framework

### DISCUSSION THREAD

**Discussion by listaction** [MaterializeInc/materialize#5353] [Q&A] [answered]
hi Team, 
thought I would ask here, what framework is used for the materialize website (the docs / quickstart ) section?

**Comment by listaction** [comment, marked as answer]
nm - for anyone looking in the future  its based on hugo :)

**Reply by benesch** [reply]
That's right! With a custom theme that you can browse here:

  * https://github.com/MaterializeInc/materialize/tree/main/doc/user/layouts/_default
  * https://github.com/MaterializeInc/materialize/tree/main/doc/user/assets/sass

**Reply by rjnn** [reply]
FYI if you would like, we can apache 2 the doc layouts/assets.

**Reply by benesch** [reply]
I actually don’t think that makes sense. The layout and style define the Materialize brand and probably ought to remain proprietary.
